stellar/deploy-contract.js

```diff
@@ -155,20 +143,21 @@ function main() {
                 .default(15)
                 .argParser(Number),
         )
-        .action((options) => {
-            mainProcessor(options, processDeploy);
+        .action((contractName, options) => {
+            printInfo('Deploying contract', contractName);
+            options.contractName = contractName;
+            mainProcessor(options, deploy);
```

Comment:
```suggestion
            mainProcessor(options, deploy, [contractName]);
```
weird to set it as a optional arg when it's positional. Pass positional args into mainProcessor which then passes to the processor directly?

Reply:
got it. updated.